src/docs/truffle/reference/truffle-commands.md

```diff
@@ -361,6 +361,8 @@ Options:
 * `<test_file>`: Name of the test file to be run. Can include path information if the file does not exist in the current directory.
 * `--compile-all`: Compile all contracts instead of intelligently choosing which contracts need to be compiled.
 * `--compile-all-debug`: Like `--compile-all`, but compiles contracts in debug mode for extra information.  Has no effect on Solidity <0.6.3.
+* `--compile-none`: Skip all contract compilations.
```

Comment:
For these two options, it might be worth adding some prose about when it is appropriate to do this and any dangerous side effects this might have.  (@eggplantzzz, can you say anything about this?)

Reply:
Oof, this is kind of a tough one. `--compile-none` is really kind of like a "you really shouldn't be doing this unless you know what you are doing"-type thing.

`--migrate-none` you could say something like, "this option could be useful if you will be deploying your own contracts and don't want Truffle to do this automatically for you".

Reply:
OK, well, for `--compile-none` we could at least put a warning saying, like, "This option is not intended for use with most tests, which will require compilation.", or something like that.

Reply:
update the doc according to the comments, thx